CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ setting is turned on DataFed will build it's libraries as shared and try to
 link to shared libraries." OFF)
 OPTION(ENABLE_END_TO_END_API_TESTS "Enable end-to-end API testing" FALSE) 
 OPTION(ENABLE_END_TO_END_WEB_TESTS "Enable end-to-end web testing with Playwright" FALSE)
+OPTION(ENABLE_FOXX_TESTS "Enable Foxx testing, off by default because it
+  will overwrite the test database." FALSE)
```

Comment:
The commands you are looking at are only partly a problem. Meaning inserting a daily message won't really cause any problems. The second line is overwriting the config document. It's possible you could execute this in prod without causing problems, but it would be risky. 

The comment is perhaps a bit unclear. My concern was with creating and destroying repos, allocations and users etc on a production database as a result of running the tests.